vlm/src/main/scala/geotrellis/contrib/vlm/DataPath.scala

```diff
@@ -0,0 +1,15 @@
+package geotrellis.contrib.vlm
+
+import java.net.{URI, URL}
+
+
+trait DataPath {
+  def path: String
+  def targetPath: String
+
+  protected def servicePrefix: String
```

Comment:
This is not part of public interface, so its not useful to use and is not used on the base trait, so for now this unnecessarily restricts the possible subclasses of `DataPath`. Either it should become somehow public and documented or be removed.